sheet.js

```diff
@@ -61,9 +61,9 @@ Sheet.prototype.generate = function() {
     row = '<x:row r="' + currRow + '" spans="1:' + cols.length + '">';
     
     for (var j = 0; j < cols.length; j++) {
-      var cellData = r[j];
+      var cellData = typeof r[j] === 'object' ? r[j].value : r[j];
```

Comment:
You've got two `typeof r[j] === 'object'` here, should be maybe be saved as `const` once above?